CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Changed:
 - Changed format of JSON protocol message sent in /set_protocols route to start a stimulation
 - Live view will always be stopped when a recording is stopped
 - Changed error message to Please turn the instrument off, unplug from the PC, and then wait 10 seconds before attempting to use again
+- Fixed heat map display bug by chaniging mean values calculation to the last 5 data points of each well.
```

Comment:
Typo: changing. Also since this is worded as a fix it should go under a `Fixed:` subsection